imqsauth/http.go

```diff
@@ -571,6 +571,10 @@ func httpHandlerLogin(central *ImqsCentral, w http.ResponseWriter, r *httpReques
 	}
 
 	if sessionkey, token, err := central.Central.Login(identity, password); err != nil {
+		if central.Central.Auditor != nil {
+			contextData := fmt.Sprintf(`{"username": "%v", "origin": "%v"}`, identity, getIPAddress(r.http))
```

Comment:
What if `identity` has a double quote inside it?

Reply:
@bmharper in that case I am just going to use the `auditUserLogAction` function like the other calls but with `0` as the `userId`.